docker-compose.archive.yml

```diff
@@ -47,6 +47,21 @@ services:
     ports:
       - "${GATEWAY_HOST}:${GATEWAY_PORT}:8000"
 
+  # localhost:9000
+  # Explorer service.
+  # It provides rich GraphQL API for querying archived data
+  archive-explorer:
+    image: subsquid/substrate-explorer:firesquid
+    environment:
```

Comment:
you can add `depends_on` with `archive-db`

Reply:
[00094e2](https://github.com/Cardinal-Cryptography/indexer/pull/6/commits/00094e2d549d4af81011689de88f7579aee82334)